packages/protocol/scripts/bash/release-on-devchain.sh

```diff
@@ -42,6 +42,7 @@ if command -v lsof; then
 fi
 
 echo "- Verify bytecode of the network"
+rm -r build/contracts
```

Comment:
Was hitting this erro without this line: https://github.com/OpenZeppelin/openzeppelin-sdk/issues/465